spec/rubocop/cop/rspec/hook_argument_spec.rb

```diff
@@ -288,5 +288,35 @@
         RUBY
       end
     end
+
+    context 'when Ruby 3.4', :ruby34, unsupported_on: :parser do
+      it 'does not flag :example for hooks' do
+        expect_no_offenses(<<~RUBY)
```

Comment:
Can you please change this to ‘it.run’ not to offend the other cop?

Reply:
I copy-pasted the current numblock specs, will change these as well.